Meshery-Adapters/consul-meshery-adapter/step1.md

```diff
@@ -7,9 +7,9 @@ After the environment has been setup, you may check up on your cluster nodes by
 
 Meshery can be downloaded, installed, deployed and auto-launched with a single command:
 
-`curl -L https://git.io/meshery | PLATFORM=docker bash -`{{execute}}
+`curl -L https://git.io/meshery | ADAPTERS=consul DEPLOY_MESHERY=true PLATFORM=docker bash -`{{execute}}
```

Comment:
We can remove `DEPLOY_MESHERY=true` as we only have case for `DEPLOY_MESHERY=false`
Adding `DEPLOY_MESHERY=true` will have no impact
```
curl -L https://git.io/meshery | ADAPTERS=consul PLATFORM=docker bash -
```

Reply:
Sure